package.json

```diff
@@ -8,6 +8,8 @@
   "scripts": {
     "build": "npm run --silent clean && babel src/server/ -d lib/server/ --ignore __tests__ && npm run --silent webpack",
     "clean": "rimraf lib public/*.worker.js public/*.bundle.js*",
+    "babel:watch": "babel src/server -d lib/server --ignore __tests__ --watch",
```

Comment:
I've been meaning to add this for a while, thanks! I do think we could extract a separate `babel` step so that we don't have to duplicate the params in `build`.